README.md

```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    RodAda Hacker is a brazillian network focused on stimulating the appropriation of new technologies by girls and women, which is based on the realization of collaborative workshops specially designed for the female audience. The name of the project is a tribute to Ada Lovelace.
```

Comment:
Although this is a literal translation of their description, to be honest, I think even in Portuguese I didn't understand what they do. I think this part "stimulating the appropriation of new technologies" is quite confuse to me (even in PT)

I found another description in their site that sounds more natural to me:
"A RodAda Hacker é uma rede que se baseia em oficinas de programação especialmente desenhadas para meninas e mulheres. Um encontro para quem quer imaginar e construir projetos incríveis e recriar as tecnologias da rede."

How about something like this:
"The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for who wants to dream, build awesome projects and rethink the technology. The name of the project is a tribute to Ada Lovelace."

Reply:
It's a great suggestion, i'll change right now.

Reply:
😄